packages/renderer/src/lib/ui/Typeahead.svelte

```diff
@@ -11,6 +11,7 @@ export let disabled: boolean = false;
 export let initialFocus: boolean = false;
 export let id: string | undefined = undefined;
 export let name: string | undefined = undefined;
+export let isRecognized: boolean = true;
```

Comment:
`isRecognized` feel very odd to me, if we want to show red, let's call it error: boolean